src/services/scheduler.ts

```diff
@@ -161,15 +161,15 @@ export async function start(dbId: string, dbUser: string) {
           if (!Modules[modulename][methodname]) {
             throw new Error(`Method ${methodname} in module ${modulename} not found`);
           }
-          output = await Modules[modulename][methodname](params);
+          const moduleCtx = (Modules[modulename] as ModuleInterface).provides?.context;
```

Comment:
This is unfortunately not everything needed. The `provides.context` of all installed modules need to be mixed together. I am pretty sure that logic lives in the apply and sync functions.

Reply:
ok, I think my latest update should do the work!